cmake/export_power_grid_model.cmake

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-cmake_minimum_required (VERSION 3.9)
+cmake_minimum_required (VERSION 3.15)
```

Comment:
Do you also need 3.15 for export target? I think for the users of C-API, we don't need this cmake version right?

Reply:
oh you're right. good call